warehouse/common/src/main/java/datawave/common/cl/OptionBuilder.java

```diff
@@ -8,16 +8,16 @@
  */
 public class OptionBuilder {
     /** Number of arguments the next Option expects; defaults to 0. */
-    public int args;
+    private int args;
```

Comment:
These changes appear to break compilation of `warehouse/query-core/src/main/java/datawave/query/cardinality/CardinalityScanner`

Reply:
Yeah -- saw that when I was cleaning up other items from here.  I added some getters/setters to the OptionBuilder, so we're good to go.